include/iarc7_fc_comms/MspConf.hpp

```diff
@@ -45,7 +45,8 @@ namespace FcComms
         static constexpr const float kMspPitchScale{10.0}; // Should be filled in with values based on (kMspMax-kMspMidpoint) / max_angle
         static constexpr const float kMspRollScale{10.0};  // Should be filled in with values based on (kMspMax-kMspMidpoint) / max_angle
         static constexpr const float kMspYawScale{10.0};   // Should be filled in with values based on (kMspMax-kMspMidpoint) / max_angle
-        static constexpr const float kMspThrottleScale{(2000.0-1000.0) / 100.0};  // (Max-Min)/100 e.g. scale 0% to 100% to min/max throttle
+        static constexpr const float kMspThrottleStartPoint{950.0};
+        static constexpr const float kMspThrottleScale{(2000.0-kMspThrottleStartPoint) / 100.0};  // (Max-Min)/100 e.g. scale 0% to 100% to min/max throttle
```

Comment:
If we are going to fix it now is the time. This is close to convention correct? If so I'll change it.